packages/components/CallToActionSection.tsx

```diff
@@ -19,12 +20,12 @@ export default function CallToActionSection() {
           <Trans
             i18nKey={t("title")}
             components={{
-              span: <span className="text-purple-400" />,
+              span: <span className={classes("text-purple-400")} />,
             }}
           />
         </Heading>
 
-        <p className="text-gray-500 max-w-3xl">{t("description")}</p>
+        <p className={classes("text-gray-500", "max-w-3xl")}>{t("description")}</p>
```

Comment:
same

Reply:
Made in: https://github.com/podcodar/webapp/pull/175/commits/5ca22732cb3405a90a8b206bc46244e34b95ba7c